sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -93,7 +93,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.6.0",
         "msrest>=0.6.10",
-        "azure-storage-blob~=12.0"
+        "azure-storage-blob>=12.3.1"
```

Comment:
shared_requirements.txt should also be updated

Reply:
Done.